tests/runtime_shell/common.ps1

```
@@ -0,0 +1,121 @@
+
+    if (-not [string]::IsNullOrWhiteSpace($StandardOutputPath)) {
+        $parameters.RedirectStandardOutput = $StandardOutputPath
+    }
+    if (-not [string]::IsNullOrWhiteSpace($StandardErrorPath)) {
+        $parameters.RedirectStandardError = $StandardErrorPath
+    }
+
+    return Start-Process @parameters
+}
+
+function Stop-FluentBit
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Stop-FluentBit' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)